fabric-recipe-api-v1/src/main/java/net/fabricmc/fabric/impl/recipe/ingredient/builtin/NbtIngredient.java

```diff
@@ -145,7 +145,7 @@ public void write(JsonObject json, NbtIngredient ingredient) {
 			json.addProperty("strict", ingredient.strict);
 
 			if (ingredient.nbt != null) {
-				json.add("nbt", NbtOps.INSTANCE.convertTo(JsonOps.INSTANCE, ingredient.nbt));
```

Comment:
Can a test for this be added

Reply:
Is it needed? The read/parse function in the same class reads the string version of Json already, (`TagParser.parseTag`) so it's backwards compatible. Not sure how can you test that any further

Reply:
I think so, to prove what the problem is and how it was solved. Would fit nicely into SerializationTests.